notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>Is it necessary to set up an endpoint? I don't see this variable used anywhere? The endpoint is set in the SDK initialization. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>yeah it was used in previous versions, forget to delete it.</p>